scylla/src/transport/session_test.rs

```diff
@@ -1435,3 +1435,38 @@ async fn test_primary_key_ordering_in_metadata() {
     assert_eq!(table.partition_key, vec!["c", "e"]);
     assert_eq!(table.clustering_key, vec!["b", "a"]);
 }
+
+#[tokio::test]
+async fn test_table_partition_in_metadata() {
```

Comment:
This test assumes that the cluster it connects to is Scylla. We running all tests with both Scylla and Cassandra in CI. The test should detect if it connected to Scylla and should skip itself if it is not.

Reply:
Is there a way to get this information from the `session` itself or should I somehow modify workflows to pass this information to the runtime (e.g. with environment variable)?

Reply:
I think that an environment variable set by the workflow is a better choice.